settings.py

```diff
@@ -12,11 +12,13 @@
 )
 
 url_mgmnt_sys = dict(
-    jira_url='https://ucsc-cgl.atlassian.net',
-    zenhub_url='https://api.zenhub.io/p1/repositories/'
+    jira_url='https://ucsc-cgl.atlassian.net/rest/api/latest/',
+    zenhub_url='https://api.zenhub.io/p1/repositories/',
+    github_url="https://api.github.com/repos/ucsc-cgp/"
```

Comment:
I don't think the GitHub URL should be in the management sysTE

Reply:
removed the github url and `repo`. added a new dict `default_orgs` to settings.py to keep this info separate from the urls. Could all the urls go into `urls` instead of being split between that and `url-mgmnt-sys`?

Reply:
We can, but it becomes a bit less precise and more difficult to read (when used in the code), IMHO for no good reason. If you have a good reason do it we can do it.

Reply:
Okay, let's leave it as is